Source/WebKit/WebProcess/WebPage/FindController.cpp

```diff
@@ -300,7 +300,7 @@ void FindController::findString(const String& string, OptionSet<FindOptions> opt
     }
 #endif
 
-    RefPtr<WebPage> protectedWebPage = m_webPage;
+    RefPtr<WebPage> protectedWebPage { m_webPage.get() };
```

Comment:
Can we create a convenience `RefPtr::RefPtr<T>(WeakPtr<T>)` constructor so we don't even have to use `get()` here?

Not required to land the patch.

Reply:
That might be nice.  Not all WeakPtr types can make a RefPtr, but we could probably do it.